src/tracer.js

```diff
@@ -202,29 +202,40 @@ function getTrimmedMetadata(eventMetadata, isRunner) {
 function getTrimmedTrace(traceSize, jsTrace) {
     let currentTraceSize = traceSize;
     const trimmedTrace = Object.assign({}, jsTrace);
-    trimmedTrace.events = jsTrace.events.sort(event => (['runner', 'trigger'].includes(event.origin) ? -1 : 1));
+    let totalTrimmedExceptions = 0;
+    let totalTrimmedEvents = 0;
+    // Trimming trace exceptions.
+    if (trimmedTrace.exceptions.length > 1) {
```

Comment:
there's no `if (currentTraceSize >= consts.MAX_TRACE_SIZE_BYTES)` here which I found a bit weird. 
does this need to happen even if currentTraceSize is small enough?

Reply:
The first check exists before the calling to this function.